numpy/add_newdocs.py

```diff
@@ -668,9 +668,8 @@ def luf(lamdaexpr, *args, **kwargs):
         will be in C-contiguous order (last-index varies the fastest).
         If order is 'F', then the returned array will be in
         Fortran-contiguous order (first-index varies the fastest).
-        If order is 'A' (default), then the returned array may be
-        in any order (either C-, Fortran-contiguous, or even discontiguous),
-        unless a copy is required, in which case it will be C-contiguous.
+        If order is 'A' (default), then the order of the returned array is
```

Comment:
@MechCoder have a look at the copy docstring of "copy" for insipration also.

"close as possible" means that the strides are sorted. If the original array is contiguous, "K" would mean that the result has the same memory layout (if a copy is made). If no copy is made "K" is equivalent to what it says for "A" currently (no order is forced for the result).